app/models/visit.rb

```diff
@@ -22,12 +25,14 @@ def validate_amount_of_adults
       errors.add :base, "There must be at least one adult visitor"
     end
     if visitors.count { |v| v.age && v.age >= adult_age } > 3
-      errors.add :base, "You can book a maximum of 3 visitors over the age of #{adult_age} on this visit"
+      errors.add :base, "You can book a maximum of 3 visitors over the age of"\
```

Comment:
I think these errors would be more appropriate on `:visitors` instead of `:base`. To localise them, it's something like this:

```ruby
errors.add :visitors, :max_3_adults, adult_age: adult_age
```

And then the appropriate localisation key will be something like:

```yaml
en:
  activemodel:
    errors:
      models:
        visit:
          attributes:
            visitors:
              max_3_adults: 'You can book a maximum of 3 visitors over the age of %{adult_age} on this visit'
```